homeassistant/components/netatmo/climate.py

```diff
@@ -279,6 +281,12 @@ def handle_event(self, event: dict) -> None:
             self.async_write_ha_state()
             return
 
+        if data["event_type"] == EVENT_TYPE_COOLING_MODE:
+            self._attr_preset_mode = NETATMO_MAP_PRESET[home[EVENT_TYPE_COOLING_MODE]]
+            self._attr_hvac_mode = HVAC_MAP_NETATMO[self._attr_preset_mode]
+            self.async_write_ha_state()
+            return
+
```

Comment:
not sure if that's ok or if we need to refactor previous if statement (as it contains also logic for presets)